packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -51,19 +55,27 @@ const defaultProps = {
   isAnimated: false,
   isInitiallyOpen: false,
   isTransparent: false,
+  isOpen: false,
 };
 
 class ToggleSectionHeader extends React.Component {
   constructor(props) {
     super(props);
 
     const { isInitiallyOpen = false } = this.props;
-    this.state = { isOpen: isInitiallyOpen };
+    this.state = { isOpen: isInitiallyOpen || this.props.isOpen };
```

Comment:
Components that support controlled and uncontrolled variations are a little tricky. There are a lot of edge cases where logic can be ambiguous.

Example:

```jsx
// Both props are supplied with different values. (A misuse of the API, but a possible)
<ToggleSectionHeader isInitiallyOpen={false} isOpen={true} />

<ToggleSectionHeader isInitiallyOpen={true} isOpen={false} />
```

```jsx
// Neither are supplied. This should probably default to uncontrolled and closed on initial render.
<ToggleSectionHeader />
```

One of the props needs to take precedence based on whether or not it is  `undefined`

Reply:
Made changes such that isOpen takes precedence if defined ed43374